ui/admin/settings.php

```diff
@@ -7,11 +7,11 @@
 			<?php
 			$default = 'tools';
 
-			$tabs = [
+			$tabs = array(
```

Comment:
:no_entry_sign: Overriding WordPress globals is prohibited. Found assignment to $tabs

<sub>hash: c21becd954a641d2a0b5eae2b4d99ec3</sub>